project/cmake/modules/FindSmbClient.cmake

```diff
@@ -23,6 +23,12 @@ find_path(SMBCLIENT_INCLUDE_DIR NAMES libsmbclient.h
 find_library(SMBCLIENT_LIBRARY NAMES smbclient
                                PATHS ${PC_SMBCLIENT_LIBDIR})
 
+find_library(TALLOC_LIBRARY talloc)
+find_library(TDB_LIBRARY tdb)
+find_library(TEVENT_LIBRARY tevent)
+find_library(WBCLIENT_LIBRARY wbclient)
+find_library(RESOLV_LIBRARY resolv)
+
```

Comment:
Are this libs required on all platforms or only on some?
It should be appended to the FPHSA call's REQUIRED_VARS as well so that cmake fails if one of the libs is not available.